hasher-matcher-actioner/scripts/populate_config_db

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+"""
+Write simple configs to the HMA config database
```

Comment:
"Sample configs"?

Reply:
Simple is intended, but perhaps not clear. Decided not to fix up the description more because I suspect that will be making another pass on this assuming the config proposal I had this morning works.